docker-compose.yml

```
@@ -36,14 +39,60 @@ services:
     image: ghcr.io/klips-project/mqm-worker/dispatcher:latest
     restart: unless-stopped
     depends_on:
-        - rabbitmq
+      - rabbitmq
     environment:
       - RABBITHOST=${RABBITMQ_HOSTNAME}
       - RABBITUSER=${RABBITMQ_DEFAULT_USER}
       - RABBITPASS=${RABBITMQ_DEFAULT_PASS}
       - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
       - WORKERQUEUE=dispatcher
 
+  error-handler:
+    build:
+      context: worker/error-handler
+    restart: unless-stopped
+    depends_on:
+      - rabbitmq
+    environment:
+      - RABBITHOST=${RABBITMQ_HOSTNAME}
+      - RABBITUSER=${RABBITMQ_DEFAULT_USER}
+      - RABBITPASS=${RABBITMQ_DEFAULT_PASS}
+      - RESULTSQUEUE=dispatcher
+      - WORKERQUEUE=DeadLetterQueue
+
+  send-mattermost-message:
+    image: ghcr.io/klips-project/mqm-worker/send-mattermost-message:latest
+    restart: unless-stopped
+    depends_on:
+      - rabbitmq
+    environment:
+      - RABBITHOST=${RABBITMQ_HOSTNAME}
+      - RABBITUSER=${RABBITMQ_DEFAULT_USER}
+      - RABBITPASS=${RABBITMQ_DEFAULT_PASS}
+      - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
+      - WORKERQUEUE=send-mattermost-message
+      - MATTERMOST_HOOK_URL=${MATTERMOST_HOOK_URL}
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
+
+  send-email:
+    image: ghcr.io/klips-project/mqm-worker/send-email:latest
+    environment:
+      - RABBITHOST=${RABBITMQ_HOSTNAME}
+      - RABBITUSER=${RABBITMQ_DEFAULT_USER}
+      - RABBITPASS=${RABBITMQ_DEFAULT_PASS}
+      - WORKERQUEUE=send-email
+      - RESULTSQUEUE=RABBITMQ_RESULTSQUEUE
```

is this missing or am I confusing something?

```suggestion
      - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
```